azure-storage-cpp-lite/src/storage_url.cpp

```diff
@@ -2,10 +2,95 @@
 
 namespace microsoft_azure {
     namespace storage {
+        bool is_alnum(char ch)        
+        {
+            return (ch >= 'A' && ch <= 'Z') || (ch >= 'a' && ch <= 'z') || (ch >= '0' && ch <= '9');
+        }
+
+        bool is_unreserved(char ch)
```

Comment:
unreserved for what? it's not very clear from the name what it's supposed to return